i.sentinel_2.ndvidiff/i.sentinel_2.ndvidiff.html

```diff
@@ -0,0 +1,71 @@
+<h2>DESCRIPTION</h2>
+
+<em>i.sentinel_2.ndvidiff</em> is a GRASS addon that calculates an NDVI difference map from Sentinel-2 data.
+
+<p>
+The goal of the addon is to compare the NDVI status from two time intervals for an AOI given by the vector map <em>aoi</em>. For each interval, <br>
+Sentinel-2 data is imported from the local file system (using <a href="https://github.com/mundialis/t.sentinel">t.sentinel.import</a>), cloud-masked (using <a href="https://github.com/mundialis/t.sentinel">t.sentinel.mask</a>),<br>
```

Comment:
```suggestion
Sentinel-2 L2A data is imported from the local file system (using <a href="https://github.com/mundialis/t.sentinel">t.sentinel.import</a>) given with` input_dir_first` and `input_dir_second`, cloud-masked (using <a href="https://github.com/mundialis/t.sentinel">t.sentinel.mask</a>),<br>
```
Maybe like this. Or make it somehow more clear, how the interval for difference calculation is given to the module. Would be nice for understanding how module works right in the beginning